src/components/IntroductionPage/index.js

```diff
@@ -0,0 +1,36 @@
+import {ButtonLink} from '@utrecht/component-library-react';
+import {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Navigate} from 'react-router-dom';
+
+import {FormContext} from 'Context';
+import Body from 'components/Body';
+import Card from 'components/Card';
+import Link from 'components/Link';
+
+const IntroductionPage = () => {
+  const {name, introductionPageContent = ''} = useContext(FormContext);
+  if (!introductionPageContent) {
+    return <Navigate replace to="startpagina" />;
+  }
+  return (
+    <Card title={name}>
+      <Body
+        modifiers={['wysiwyg']}
+        component="div"
+        dangerouslySetInnerHTML={{__html: introductionPageContent}}
+      />
+
+      <Link to="/startpagina" component={ButtonLink} appearance="primary-action-button">
+        <FormattedMessage
+          description="Button text for link to continue from introduction page to start page"
+          defaultMessage="Continue ➜"
```

Comment:
Should this be inside the text and not an icon?

Reply:
I wanted to avoid discussion since the design includes it - I think at some point someone will complain about it and then we will need a mechanism to optionally set an icon, which is a complicated thing so went for the simple route for now.